lib/Compiler.cpp

```diff
@@ -571,6 +571,9 @@ int RunPassPipeline(llvm::Module &M, llvm::raw_svector_ostream *binaryStream) {
     // This pass needs to be after every inlining to make sure we are capable of
     // removing every addrspacecast. It only needs to run if generic addrspace
     // is used.
+
+    pm.addPass(clspv::ReplaceLLVMIntrinsicsPass());
+
     if (clspv::Option::LanguageUsesGenericAddressSpace()) {
```

Comment:
I added it outside because it helps in general, and I thought it's somewhat a big change for it to just change due to generic address space.
I moved it more to the top of the pipeline with a comment

Reply:
Could you provide a test showing how it helps in general, it's not clear to me looking at the modified tests. `test/LLVMIntrinsics/memcpy_from_constant.cl` even tends to say that it is not helping.